SW4STM32/BellaLui/Application/HostBoard/Inc/misc/state_machine_helpers.h

```diff
@@ -0,0 +1,34 @@
+#ifndef MISC_STATE_MACHINE_HELPERS_H_
+#define MISC_STATE_MACHINE_HELPERS_H_
+
+#include <stdint.h>
+
+namespace state_machine_helpers {
+    // TODO: change to Enum
```

Comment:
Yeah the only reason I didn't do it right away is because I couldn't compile state_machine.c so I kept everything as simple as possible. Once we can compile it we can start with this quick change.